src/main/java/com/yahoo/identity/services/account/Account.java

```diff
@@ -21,6 +21,8 @@
     @Nonnull
     String getEmail();
 
+    boolean getEmailStatus();
```

Comment:
Maybe `isEmailVerified` would be better?

Reply:
Sorry for the ambiguity caused. Fixed.